### ISSUE TITLE
UPDATE: need to update the SampleData module 

### ISSUE THREAD

**Issue by Punzo** [Punzo/SlicerAstro#112]
Data are not anymore available at the links provided. The best would be to put them on github and retrieve them from there.

**Comment by vdhulst** [comment]
looks like the entire slicer.kitware.com site is in disarray. I can not even get an overview of all existing extensions. This may have to do with the new Kitware website which was launched some time early December 2021. They should have been backward compatible, so this may be a bug on their side.

**Comment by Punzo** [comment]
yes, but I am not sure they will provide any solution. It looks like that also the data that we used as sample data are lost. We would need to re-get them from the original source.

**Comment by vdhulst** [comment]
I can easily recover all of the files. Some of them I have at hand.
